core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
I just recently moved `Fix` _out_ of this package, to discourage users from specializing on a fixed-point type. I don’t think it should be brought back in.

The idea being that `Fix` (or `Mu` or `Nu`) should not be referenced in most of the places you use Matryoshka.